src/js/themes/media-theme.js

```diff
@@ -1,28 +1,26 @@
 class MediaTheme extends HTMLElement {
-  constructor(template = ``, options = {}) {
+  static template = '';
+
+  constructor() {
     super();
+    this.attachShadow({ mode: 'open' });
+  }
 
-    options = Object.assign(
-      {
-        // Default options
-      },
-      options
-    );
+  connectedCallback() {
+    this.render();
+  }
 
-    // Expose the template publicaly for other uses
-    this.template = template;
+  render() {
+    this.shadowRoot.textContent = '';
```

Comment:
any reason this is necessary?

Reply:
yes, to clear the existing DOM nodes if this.render() would be called more than once.

Reply:
ok cool that makes perfect sense and is a hard problem longer term. still non-blocking but we'll probably need to work this out if each re-render will completely blow away the dom and re-write it from scratch. May need separate methods here for "initializing" the shadowRoot vs. "re-rendering" (may not, just thinking about an architecture for "updates over time" esp. in relation to DOM element instances and e.g. our template-parts architecture for player.